### PR TITLE
Update README.md - Example of default retry to all ajax requests.

### PR DESCRIPTION
**Pull request by Aminadav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,21 @@ $.ajax({
     });
 ```
 
+## Allow default to all ajax requests
+
+In this example, there is default retry to all ajax requests, for 5 seconds between retries
+
+      $.ajaxSettings.
+            shouldRetry= function() {
```
Comment (dcherman) [inline README.md]: Please put this on the above line, and we're missing a space after `=`

File: README.md [hunk 2]
```diff
@@ -49,6 +49,21 @@ $.ajax({
...
+                // Always retry this request with a delay of 250ms between retries
+                return $.Deferred(function(dfr) {
+                    setTimeout(function() {
+                        dfr.resolve(true);
+                    }, 5000);
+                }).promise();
+            }
```
Comment (dcherman) [inline README.md]: The indentation of this should be such that it lines up with the `$`, and we're missing a `;` after the closing brace

**Review by dcherman — CHANGES_REQUESTED** [review]
Sorry for the (super) long delay here.  If you'd still like to merge this, I'd be happy to do so, but we need to update it to maintain the same code style as is in the rest of `README.md`.  Thanks for the contribution!

File: README.md [hunk 3]
```diff
@@ -49,6 +49,21 @@ $.ajax({
     });
 ```
 
+## Allow default to all ajax requests
```
Comment (dcherman) [inline README.md]: Please update this to `Add retry logic for all ajax requests`

File: README.md [hunk 4]
```diff
@@ -49,6 +49,21 @@ $.ajax({
...
+
+In this example, there is default retry to all ajax requests, for 5 seconds between retries
```
Comment (dcherman) [inline README.md]: Please update this to `Add logic to retry all failed requests with a 5 second delay`

**Review by dcherman — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Aminadav) [reply README.md:64]: Fixed

File: README.md (hunk 1 above)
Comment (Aminadav) [reply README.md:57]: Fixed

File: README.md (hunk 4 above)
Comment (Aminadav) [reply README.md:54]: Fixed

File: README.md (hunk 3 above)
Comment (Aminadav) [reply README.md:52]: Fixed

**Comment by Aminadav** [comment]
@dcherman All fixed.

**Pull request closed without merging**